### PR TITLE
[ISSUE#1881][MAS2.1.2][Keyboard Navigation- Welcome Tab screen] Focus gets trapped on first and last element of the page when user navigates through the page

### PR DESCRIPTION
**Pull request by denscollo**
Solves #1881

### Description
Fixes how the focus gets trapped between the last and the first element of the pane.
 
### Changes made
We added a new condition to the **_EventHandlers_** class which will manage the focus when the emulator is running in _macOS_. Also, we included two auxiliary methods to find the first and last elements of the active pane and be able to move the focus between them.

Additionally, we added two unit tests to cover the new _keydown_ event.

### Testing
In the following image, you can see how the focus goes from the last element of the pane to the first when navigating with Tab key and vice-versa when using Shift+Tab navigation: 

FocusTrap

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - [1934](https://github.com/microsoft/BotFramework-Emulator/pull/1934)
   - [1935](https://github.com/microsoft/BotFramework-Emulator/pull/1935)
   - [1936](https://github.com/microsoft/BotFramework-Emulator/pull/1936)
+  - [2001](https://github.com/microsoft/BotFramework-Emulator/pull/2001)
```
Comment (tonyanziano) [inline CHANGELOG.md]: this needs to be under the "Unreleased" header and not the "4.6.0" header

Comment (denscollo) [reply CHANGELOG.md]: Thanks for noticing! I just updated the changelog entry.

File: packages/app/client/src/utils/eventHandlers.ts [hunk 2]
```diff
@@ -121,7 +135,25 @@ class EventHandlers {
         } as Notification);
       }
     }
+
+    if (isMac()) {
+      const tabPressed: boolean = key === 'tab' || keyCode === 9;
```
Comment (corinagum) [inline packages/app/client/src/utils/eventHandlers.ts]: I believe keyCode is deprecated. Can we remove the reference? key === tab should be sufficient.

File: packages/app/client/src/utils/eventHandlers.ts [hunk 3]
```diff
@@ -121,7 +135,25 @@ class EventHandlers {
...
+      const lastDecendants = EventHandlers.getLastDecendants(document.querySelector('main'));
+      const firstElement = document.querySelector('nav').firstElementChild as HTMLElement;
```
Comment (corinagum) [inline packages/app/client/src/utils/eventHandlers.ts]: move to line 152

File: packages/app/client/src/utils/eventHandlers.ts [hunk 4]
```diff
@@ -121,7 +135,25 @@ class EventHandlers {
...
+      const lastElement = lastDecendants[lastDecendants.length - 1] as HTMLElement;
```
Comment (corinagum) [inline packages/app/client/src/utils/eventHandlers.ts]: move to line 149

File: packages/app/client/src/utils/eventHandlers.ts [hunk 5]
```diff
@@ -121,7 +135,25 @@ class EventHandlers {
         } as Notification);
       }
     }
+
+    if (isMac()) {
+      const tabPressed: boolean = key === 'tab' || keyCode === 9;
+      const lastDecendants = EventHandlers.getLastDecendants(document.querySelector('main'));
```
Comment (corinagum) [inline packages/app/client/src/utils/eventHandlers.ts]: Move to line 149

**Comment by denscollo** [comment]
@corinagum if we move lines 142 & 143 to the ones that you told us, the `isFirstElement` and `isLastElement` variables are not going to work. If you want us to move those line, we need to update the if clause to be something like: 
`      if (shiftPressed && document.activeElement === document.querySelector('nav').firstElementChild as HTMLElement) {
    }`

And something similar would happen to the other if clause, since it uses lastDecendants to obtain the lastElement.

Let us know if you want us to proceed with this change.

**Pull request merged**